Added translation for dutch and lang param in module

amavis442 opened the pull request:
Without $lang in module yii can gives an error. Also added translation for dutch.

bupy7 commented on messages/nl/core.php:
```diff
@@ -0,0 +1,27 @@
+ * Language 'EN'.
+ * 
+ * @author Vasilij Belosludcev http://mihaly4.ru
+ * @version 1.0.0
```
new version will 1.1.1 =)

bupy7 commented on messages/nl/core.php:
```diff
@@ -0,0 +1,27 @@
+/**
+ * Language 'EN'.
+ * 
+ * @author Vasilij Belosludcev http://mihaly4.ru
```
You author, not me.

bupy7 commented on messages/nl/core.php:
```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Language 'EN'.
```
Language is 'NL'

bupy7 commented on Module.php:
```diff
@@ -76,6 +76,8 @@ class Module extends \yii\base\Module
      */
     public $urlToFiles;
     
+    public $lang;
```
where this uses? i not understand.

amavis442 commented:
Corrected copyright notice and version number. Reverted $lang in module. I used $lang in an override controller for ManageController so i could change the view _form with an extra field for an menu item. My apologies for this inconvenience.

bupy7 commented:
Thanks!
